lib/fs.js

```diff
@@ -2138,3 +2138,24 @@ SyncWriteStream.prototype.destroy = function() {
 };
 
 SyncWriteStream.prototype.destroySoon = SyncWriteStream.prototype.destroy;
+
+fs.mkdtemp = function(template, callback) {
```

Comment:
I've done it this way because other functions do it like this, see e.g. `access`.